docs/FreeSWITCH-Explained/Codecs-and-Media/Video-Codecs_1048673.mdx

```diff
@@ -1,24 +1,12 @@
 
-# Video Codecs 
-
- 
+# Video Codecs
 
 ## Pass-through video codecs
 
-* H261 - H.261 Video
-* H263 - H.263 Video
-* H263-1998 - H.263-1998 Video
-* H263-2000 - H.263-2000 Video
-* H264 - H.264 Video  
-   * Provided by mod\_h26X.
-* Theora passthrough.  
-   * Provided by mod\_theora.
-* MP4 Video passthrough.  
-   * Provided by mod\_mp4v.
-
-### Comments:
-
-| H264 now provided by mod\_av which use ffmpeg.also there are VP8, VP9 provided by vpx ![](/images/icons/contenttypes/comment_16.png) Posted by niuzb at Sep 09, 2020 21:45 |
-| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-
+* H264 - H.264 Video
+   * Provided by mod\_av.
+* VP8
+   * Provided by mod\_vp8
```

Comment:
Is build in like VP9

<img width="292" height="46" alt="Image" src="https://github.com/user-attachments/assets/5df8f737-298e-4319-abb8-f4eab5043c82" />